### DISCUSSION TITLE
Is there a place to track the progress of Parcel 2 ?

### DISCUSSION THREAD

**Discussion by RPDeshaies** [parcel-bundler/parcel#5356] [Q&A] [answered]
Now that the Github project has been closed, I was wondering where we could follow the progress on Parcel 2 or help with latest pre-releases.

Thanks!

**Comment by DeMoorJasper** [comment, marked as answer]
To help out, you can have a look at all issues that have been marked Parcel 2: https://github.com/parcel-bundler/parcel/issues?q=is%3Aopen+is%3Aissue+label%3A%22%E2%9C%A8+Parcel+2%22

There's currently no way to track the Parcel 2 progress, but we're very close to a release candidate, we have to finalise the api and fix some remaining major bugs in cache, scope hoisting and source maps. I'm hopeful we can get a release candidate out by january. For now the nightlies are already fairly stable, might even be more reliable than Parcel 1 at this point.

**Reply by RPDeshaies** [reply]
Very nice to hear 😄

**Reply by john01dav** [reply]
Parcel 2 is nowhere near ready to release with error messages as useless as [this](https://github.com/parcel-bundler/parcel/discussions/5385). I *hope* that I'm missing something and there's a way to be productive in it, but I don't see any evidence of that right now.

**Reply by DeMoorJasper** [reply]
@john01dav we appreciate bug reports but don't spam other issues just to highlight a single bug that we're not aware of or can't seem to reproduce...

If you wanna be helpful and not see the error debug it or at least do a proper bug report preferably with some way to reproduce this reliably.

**Reply by john01dav** [reply]
> @john01dav we appreciate bug reports but don't spam other issues just to highlight a single bug that we're not aware of or can't seem to reproduce...
> 
> If you wanna be helpful and not see the error debug it or at least do a proper bug report preferably with some way to reproduce this reliably.

Yeah, you're right. Sorry about that. I'm just frustrated. I'll try to make the best bug reports that I can, but I don't think that they'll be particularly good given the progress that I've been able to make with debugging.
